addons/hivedr/src/main/java/org/apache/falcon/hive/util/FileUtils.java

```diff
@@ -34,8 +34,10 @@
 public final class FileUtils {
 
     public static final String DEFAULT_EVENT_STORE_PATH = StringUtils.removeEnd(DRStatusStore
-            .BASE_DEFAULT_STORE_PATH,  File.separator) + File.separator + "Events";
+            .BASE_DEFAULT_STORE_PATH,  File.separator) + File.separator + "Events" + File.separator;
```

Comment:
I see you try to append File.separator if it is not at the end of the string. StringUtils::appendIfMissing would be better to use in this case. Same for the other place in